recipe/meta.yaml

```
@@ -68,9 +82,11 @@ requirements:
     - matplotlib-base
     - h5py
     - python
+    - {{ mpi }}  # [mpi != 'nompi']
   run:
-    - dagmc 3.2.1  # [dagmc == 'dagmc']
+    - dagmc * 3.2.1 * {{ mpi_prefix }}_*  # [dagmc == 'dagmc']
```

should this line be the same as the dagmc line in the host section
```suggestion
    - dagmc 3.2.1 {{ mpi_prefix }}_*  # [dagmc == 'dagmc']
```